gbfs-validator/cli.js

```diff
@@ -1,17 +1,95 @@
+#!/usr/bin/env node
+
 const { inspect } = require('util')
-const GBFS = require('./')
+const commander = require('commander')
+const fs = require('fs')
+const fsPath = require('path')
+const GBFS = require('./gbfs')
+const pjson = require('./package.json')
+
+getFeedValidationReport = async (url) => {
+  const gbfs = new GBFS(url)
+  return gbfs.validation()
+}
+
+const isExitOverrided = () => (process.env.EXIT_OVERRIDE === 'true')
 
-if (!process.argv[2]) {
-  console.error('Usage: gbfs-validator GBFS_URL')
-  process.exit(1)
+const printingReport = (options) => (options.printReport === 'yes')
+
+const exitProcess = (code) => {
+  if (!isExitOverrided && code === 1) {
+    process.exit(code)
+  }
+  process.exit(0)
 }
 
-const gbfs = new GBFS(process.argv[2])
+parseOptions = () => {
+  commander
+    .version(pjson.version, '-v, --version')
+    .usage('[OPTIONS]...')
+    .requiredOption('-u, --url <feed_url>', 'URL of the GBFS feed')
+    .option('-vb, --verbose', 'Verbose mode prints debugging console logs')
+    .option('-s, --save-report <report_path>', 'Local path to output report file')
+    .addOption(new commander.Option('-pr, --print-report <yes_no>', 'Print report to standard output').default('yes').choices(['yes', 'no']))
```

Comment:
For consistency with `-s`, could we use one letter `-p` for printing?

```suggestion
    .addOption(new commander.Option('-p, --print-report <yes_no>', 'Print report to standard output').default('yes').choices(['yes', 'no']))
```

Reply:
I connected with @richfab, and we couldn't find any strong feelings either way.  Let's keep it as it is for simplicity; in the future, more options will be added, and the short argument name will be always a point of conflict.